_opensearch/query-dsl/query-filter-context.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Query and filter context
+parent: Query DSL
+nav_order: 5
+---
+
+# Query and filter context
+
+Queries consist of query clauses, which can be run in a [_filter context_](#filter-context) or [_query context_](#query-context). A query clause in a filter context asks the question "_Does_ the document match the query clause?" and returns matching documents. A query clause in a query context asks the question "_How well_ does the document match the query clause?" and returns matching documents, providing the relevance of each document in the [_relevance score_](#relevance-score).
```

Comment:
RE: "A query clause in a filter context asks the question "_Does_ the document match the query clause?" and returns matching documents."
Does a query clause return documents regardless of the answer? In other words, will it return documents even when the answer is "No"?

Suggestion only:
A query clause in a query context asks the question "_How well_ does the document match the query clause?", returns matching documents, and provides the relevance of each document in the [_relevance score_](#relevance-score). (or some variation on that.)

Reply:
No, in a filter context, if the document does not match, it is not returned.